changelogs/unreleased/5508-move-increment-calculation-to-release-version-pipeline.yml

```diff
@@ -0,0 +1,6 @@
+description: Move increment calculation to release_version pipeline.
```

Comment:
Don't we want to have a changelog entry for this?

Reply:
Probably a minor at best

Reply:
Then we should exclude iso5, but I don't think that's an issue.